packages/@okta/vuepress-site/guides/sign-into-spa/section01/index.md

```diff
@@ -2,25 +2,28 @@
 title: Before You Begin
```

Comment:
@swiftone  The Guides template should display the title of each section automatically at the top of the page. 
@alexdahl-okta Did we decide that we wanted to also include the name of the guide itself? I think you mocked up "Sign Users into Your Single-Page Apps: Before You Begin"

Reply:
Yeah, last we had was a hierarchy like:
<img width="433" alt="title" src="https://user-images.githubusercontent.com/26014482/55663811-299c7400-57bf-11e9-9810-2704f03cd748.png">

That is, a ~body-font-sized-but-bold guide title, followed by an h1 step title.

(IMO the guide title should be included programmatically, so that it is not repeated in the content of a dozen steps.)